cumulusci.yml

```
@@ -538,13 +546,15 @@ flows:
                 flow: config_managed
             4:
                 task: enable_course_connections
-            5:
+            5: 
+                task: add_citizenship_status_values
```

This will accomplish calling the task from EDA defined as eda in this yml file.
```suggestion
                task: eda:add_citizenship_status_values
```